plugins/devlink/meson.build

```diff
@@ -0,0 +1,63 @@
+if host_machine.system() == 'linux'
+  cargs = ['-DG_LOG_DOMAIN="FuPluginDevlink"']
+  plugins += {meson.current_source_dir().split('/')[-1]: true}
```

Comment:
take a look at https://github.com/fwupd/fwupd/commit/ce80dbb93af6e8a08f95974135043fc903d23b98 and port your plugin to use it please.